openc3/bin/openc3cli

```diff
@@ -79,7 +79,7 @@ def print_usage
   puts "  cli bridgesetup CONFIG_FILENAME   # Create a default config file"
   puts "  cli pkginstall PKGFILENAME SCOPE  # Install loaded package (Ruby gem or python package)"
   puts "  cli pkguninstall PKGFILENAME SCOPE  # Uninstall loaded package (Ruby gem or python package)"
-  puts "  cli rubysloc                      # Counts Ruby SLOC recursively. Run with --help for more info."
+  puts "  cli rubysloc                      # DEPRECATED: Please use scc (https://github.com/boyter/scc)"
```

Comment:
I don't think we should support because scc is so much better and we don't even support Python